### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Update content to match the copy doc
- Add prettier config file
- Make tab heading reveal correctly
- Improve small screen layout 

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the content matches [the copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#)
- Check small screen view is better
- Ensure thhe tab heading "Click for install steps" works as you would expect

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/7682

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (steverydz) [inline .prettierrc:1]: We could remove this file if it's empty

File: package.json [hunk 2]
```diff
@@ -44,7 +44,8 @@
   "devDependencies": {
     "concurrently": "5.1.0",
     "stylelint": "13.2.1",
+    "stylelint-config-prettier": "^8.0.1",
```
Comment (steverydz) [inline package.json:47]: Can we pin this dependency? e.g. remove `^`

File: .prettierrc (hunk 1 above)
Comment (anthonydillon) [reply .prettierrc:1]: A lot of people set there IDE to format on save if a prettier file is in the project.

File: package.json (hunk 2 above)
Comment (anthonydillon) [reply package.json]: Good point, done.

**Comment by waynecrosby** [comment]
@anthonydillon Can you please increase the size of the 'Zero-Ops infrastructure' illustration to w351 x h200 and reduce the height of the 'Clusters for scale' illustration to h200px. Otherwise looks good! Thanks

**Comment by meralonso** [comment]
@anthonydillon windows logo looks a bit pixelated - could you please replace png for [this svg](https://assets.ubuntu.com/v1/11ecbd2d-2018-logo-windows.svg)? 

And the native installer as well, please try [this one](https://assets.ubuntu.com/v1/6471bdc4-native-installer-win-mac.svg)

**Comment by anthonydillon** [comment]
@meralonso that windows logo has a lot of white space around it.

**Comment by anthonydillon** [comment]
@waynecrosby done.
@meralonso I have updated the native image but the windows one will need the whitespace stripped from it unfortunately.

**Comment by waynecrosby** [comment]
@anthonydillon Thanks. Here is a Windows logo with the space removed:
https://assets.ubuntu.com/v1/429929de-2020-logo-windows.svg

**Comment by meralonso** [comment]
Thanks @anthonydillon and @waynecrosby - the rest looks good to me

**Comment by SirSamTumless** [comment]
@anthonydillon Why's this got a don't merge label?

**Comment by anthonydillon** [comment]
I'm going to use it for the masterclass later

**Pull request merged**